include/config_defaults.h

```diff
@@ -203,6 +203,12 @@
 #ifndef GAME_BRICK_BREAKER
 #define GAME_BRICK_BREAKER 0
 #endif
+#ifndef TOOL_FLASHLIGHT
+#define TOOL_FLASHLIGHT 1
+#endif
+#ifndef TOOL_CALCULATOR
+#define TOOL_CALCULATOR 1
```

Comment:
The default value by `0`, please? They will activate freely depending on who is needed.